Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "minireq"
-version = "0.1.0"
```

Comment:
`0.1.1` as in the changelog, no?

Reply:
Ah woops, thanks for catching this. I was going to do 0.2.0 then realized it's fine as a patch. Will update